lib/core/test/unit/Cardano/Wallet/DB/SqliteSpec.hs

```diff
@@ -646,8 +647,8 @@ testCpSeq = snd $ initWallet block0 genesisParameters initDummyStateSeq
 initDummyStateSeq :: SeqState 'Testnet ShelleyKey
 initDummyStateSeq = mkSeqState (xprv, mempty) defaultAddressPoolGap
   where
-      bytes = entropyToBytes <$> unsafePerformIO $ genEntropy @(EntropySize 15)
-      xprv = Seq.generateKeyFromSeed (Passphrase bytes, mempty) mempty
+      mw = SomeMnemonic $ unsafePerformIO (generate $ genMnemonic @15)
```

Comment:
Ok to call a `SomeMnemonic` `mw` as in "mnemonic words"?

We have used the `mw` name in the past, I believe.